src/state-store.js

```diff
@@ -10,10 +10,12 @@ class StateStore {
         db.createObjectStore('states')
       }
       dbOpenRequest.onsuccess = () => {
+        this.isConnected = true
```

Comment:
Kinda weird but we have a tradition of only ever accessing internal object state via methods, which would imply `isConnected` should be a method. We could consider changing that because ES6 has a convenient facility for defining getters, but originally it was to avoid confusion about which properties were public because of the lack of privacy controls in JS.